ocaml/xapi/export.ml

```diff
@@ -107,6 +107,19 @@ let rec update_table ~__context ~include_snapshots ~preserve_power_state ~includ
 				end)
 			vm.API.vM_VGPUs;
 
+		(* add all PVS proxies that have a VIF belonging to this VM, add their
+		 * PVS sites as well
+		 *)
+		Db.PVS_proxy.get_all_records ~__context
+		|> List.filter (fun (_,p)-> List.mem p.API.pVS_proxy_VIF vm.API.vM_VIFs)
+		|> List.iter
+			(fun (ref, proxy) ->
+				if Db.is_valid_ref __context ref then begin
+					add ref;
+					add proxy.API.pVS_proxy_site;
+					add proxy.API.pVS_proxy_VIF;
```

Comment:
We can rely on the VIF to be there already, because it is added above. That is not true for the other objects.